ui/scss/component/_markdown-preview.scss

```diff
@@ -13,7 +13,29 @@
     font-size: inherit;
     font-weight: var(--font-weight-bold);
     margin-bottom: var(--spacing-medium);
-    padding-top: var(--spacing-medium);
+
+    &:not(:first-child) {
+      margin-top: var(--spacing-large);
+    }
+  }
+
+  h1 {
+    font-size: 1.8em;
+  }
+  h2 {
+    font-size: 1.7em;
+  }
+  h3 {
+    font-size: 1.6em;
+  }
+  h4 {
+    font-size: 1.5em;
+  }
+  h5 {
+    font-size: 1.4em;
+  }
+  h6 {
```

Comment:
h5 and h6 can probably be closer to 1.0, this would also allow a larger gradation between them

since they are bold, h6 could probably go all the way down to 1.0